src/KingSystem/Event/evtInfoData.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <KingSystem/Resource/resLoadRequest.h>
+#include <KingSystem/Utils/Byaml/Byaml.h>
+#include <heap/seadDisposer.h>
+#include <prim/seadSafeString.h>
+
+namespace ksys::evt {
+
+class InfoDataUnknownType {
+public:
+    virtual ~InfoDataUnknownType();
+};
+
+class InfoData {
+    SEAD_SINGLETON_DISPOSER(InfoData)
+    InfoData();
+    virtual ~InfoData();
+
+public:
+    bool getEntry(al::ByamlIter* iter, const sead::SafeString& eventFlowName,
```

Comment:
```suggestion
    bool getEntry(al::ByamlIter* iter, const sead::SafeString& event_flow_name,
```

Nintendo uses normal_snake_case for non-member variables